### DISCUSSION TITLE
Inertia V2 Support

### DISCUSSION THREAD

**Discussion by MattBromley98** [RobertBoes/inertia-breadcrumbs#21] [Ideas]
With Inertia V2 Beta out, would be good to get this package upgraded to be compatible with the changes made there.

**Comment by MattBromley98** [comment]
Looks like it should be a simple upgrade of a dependency when it gets released! Seems to work great in my project! :)

**Reply by RobertBoes** [reply]
Haven't looked into it yet, but it should be compatible as-is, just the version requirement would stop it from installing.

Not sure if it's worth it, but also wanted to look into the new features, such as allowing the breadcrumbs to be deferred

**Reply by MattBromley98** [reply]
That sounds exciting - could definitely have it as a toggle in the config, just as some users might use it in their structured markup (BreadcrumbList) and want it passing through immediately in SSR for SEO.
